src/YiiElasticSearch/MultiSearchDataProvider.php

```diff
@@ -0,0 +1,133 @@
+<?php
+namespace YiiElasticSearch;
+
+/**
+ * A Data provider for searching into multiple indices
+ */
+class MultiSearchDataProvider extends DataProvider
+{
+    public $connID = 'elasticSearch';
+    public $models = array();
```

Comment:
@mikehaertl I'm settings those in the class constructor so yeah they are expected. Regarding the reflection, it seems odd I admit but If i'm remembering correctly (it's been a while), i want to emulate the `ClassName::class` and get the full namespaced class path for older php version >=5.3.